src/mscorlib/shared/System/StringSpanHelpers.cs

```diff
@@ -125,5 +125,38 @@ public static ReadOnlySpan<char> Remove(this ReadOnlySpan<char> source, int star
             source.Slice(startIndex + count).CopyTo(result.Slice(startIndex));
             return result;
         }
+
+        // Returns the index of the last occurrence of a specified character in the current instance.
```

Comment:
This comment applies to this overload of LastIndexOf:
`LastIndexOf(this ReadOnlySpan<char> source, char value, int startIndex, int count)`